Sources/Presentr/CoverHorizontalAnimation.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import CoreGraphics
```

Comment:
Nit: Should we order these imports alphabetically?

Reply:
I guess we should, normally swift format does this automatically in the main app

Reply:
I will leave this for now if thats cool